cmd/delete/delete.go

```diff
@@ -36,8 +36,8 @@ If you supply the |--rmdirs| flag, it will remove all empty directories along wi
 You can also use the separate command |rmdir| or |rmdirs| to
 delete empty directories only.
 
-For example, to delete all files bigger than 100MBytes, you may first want to check what
-would be deleted (use either):
+For example, to delete all files bigger than 100 mebibytes, you may first want to
```

Comment:
Not a fan of `mebibytes`! I'd prefer 100 MiB or 100 MiByte I think.

What do you think?

Reply:
I agree. Failed attempt at writing a bit more more "natural language".. :) Like MiB most, as that seems to be *the* standard unit prefix.